hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/UserGroupInformation.java

```diff
@@ -91,6 +89,7 @@
 import org.apache.hadoop.util.StringUtils;
 import org.apache.hadoop.util.Time;
 
+import org.apache.hadoop.util.subject.SubjectAdapter;
```

Comment:
this "import" belongs to the above category

Reply:
moved up in the latest commit